web/src/settings_config.ts

```diff
@@ -888,6 +892,7 @@ export const all_notifications = (settings_object: Settings): AllNotifications =
                 followed_topic_notification_settings,
                 settings_object,
             ),
+            help_link: $t({defaultMessage: "/help/follow-a-topic"}),
```

Comment:
You may want to consider not marking the link for translation.

Reply:
Thanks for your suggestion, @Aditya8840! I appreciate it. I’ve unmarked the help link from translation as recommended.